sdk/keyvault/azure-security-keyvault-keys/test/key_client_test_live.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#if defined(_MSC_VER)
+#define _CRT_SECURE_NO_WARNINGS
+#endif
+
+#include "gtest/gtest.h"
+
+#include "key_client_base_test.hpp"
+
+#include <azure/keyvault/key_vault.hpp>
+
+#include <string>
+
+using namespace Azure::Security::KeyVault::Keys::Test;
+
+TEST_F(KeyVaultClientTest, GetKey)
+{
+  Azure::Security::KeyVault::Keys::KeyClient keyClient(m_keyVaultUrl, m_credential);
+  // Assuming and RS Key exists in the KeyVault Account.
+  std::string keyName("testKey");
```

Comment:
This is a good start.  We need to increase the coverage so that we cover encoded values and unicode characters etc.  Ensure that values store and round trip as expected.